content/en/cloudcraft/advanced/auto-layout-via-api.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Automate Snapshots of Cloud Accounts via the Cloudcraft API
+kind: guide
+---
+
+## Overview
+
+Cloudcraft's **Auto Layout** feature, accessible through the web application, is a powerful tool for automatically generating diagrams of your AWS environment. This functionality can significantly streamline documentation processes and facilitate the onboarding of new team members.
+
+This guide provides a step-by-step approach to utilizing this feature via common command-line utilities and the Cloudcraft developer API.
+
+<div class="alert alert-info">The ability to add and scan AWS accounts, as well as to use Cloudcraft's developer API, is only available to Pro subscribers. Check out <a href="https://www.cloudcraft.co/pricing">Cloudcraft's pricing page</a> for more information.</div>
+
+## Prerequisites
+
+- An active [Cloudcraft Pro subscription][1].
+- An API key with read-write permissions.
+- Access to a Unix-like environment (Linux, macOS, or Windows Subsystem for Linux).
+- Familiarity with command-line operations.
+- Basic knowledge of API usage.
+
+## Take a snapshot of the account
```

Comment:
Do we need to add a step here to get the AWS / Azure account id using list accounts API and identify the right value for ACCOUNT_ID?

Since these ACCOUNT_IDs generated by Cloudcraft, the user will not know this offhand.

Reply:
I think that should be its own article, which we need anyway. This come up on support all the time because of access control.

Reply:
Should we alteast then mention that in the pre-requisites - you should have the Cloudcraft issued identifier for the interested AWS account handy?

Reply:
Added on [c672e98](https://github.com/DataDog/documentation/pull/22262/commits/c672e98c3afa08a4c355494a46ef046643ef83cc).